src/banking_stage.rs

```diff
@@ -79,7 +79,10 @@ impl BankingStage {
     ) -> Result<()> {
         let timer = Duration::new(1, 0);
         let recv_start = Instant::now();
-        let mms = verified_receiver.recv_timeout(timer)?;
+        let mut mms = verified_receiver.recv_timeout(timer)?;
+        while let Ok(mut nq) = verified_receiver.try_recv() {
```

Comment:
Can you add a utility function for this pattern? And update the other locations. Looks like you use this pattern regularly.

Reply:
If we required 100% branch coverage, adding that utility function would be the laziest path forward. :)